.buildkite/scripts/stack_tests_windows.ps1

```diff
@@ -0,0 +1,68 @@
+$ErrorActionPreference = "Stop" # set -e
+
+# Forcing to checkout again all the files with a correct autocrlf.
+# Doing this here because we cannot set git clone options before.
+function fixCRLF {
+    Write-Host "-- Fixing CRLF in git checkout --"
+    git config core.autocrlf input
+    git rm --quiet --cached -r .
+    git reset --quiet --hard
+}
+
+function withGolang($version) {
+    # Avoid conflicts with previous installations.
+    Remove-Item env:GOROOT
+
+    Write-Host "-- Install golang $version --"
+    choco install -y golang --version $version
+    setupChocolateyPath
+    go version
+    go env
+}
+
+function withDocker($version) {
+    Write-Host "-- Install Docker $version --"
+    choco install -y Containers Microsoft-Hyper-V --source windowsfeatures
+    choco install -y docker-engine --version $version
+    choco install -y docker-cli --version $version
+    setupChocolateyPath
+}
+
+function withDockerCompose($version) {
+    Write-Host "-- Install Docker Compose $version --"
+    choco install -y docker-compose --version $version
+    setupChocolateyPath
+}
+
+function setupChocolateyPath() {
+    $env:ChocolateyInstall = Convert-Path "$((Get-Command choco).Path)\..\.."
+    Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
+    refreshenv
+}
+
+
+fixCRLF
+
+withGolang $env:GO_VERSION
+withDocker $env:DOCKER_VERSION
+withDockerCompose $env:DOCKER_COMPOSE_VERSION
+
+Write-Host "--- Docker Info"
+docker info
+
+echo "--- Downloading Go modules"
+go version
+go mod download -x
+
+echo "--- Running stack tests"
+go version
+$ErrorActionPreference = "Continue" # set +e
```

Comment:
Here it will be shown the same go version as in line 54, no ?

If so, this could be removed.

```suggestion
echo "--- Running stack tests"
$ErrorActionPreference = "Continue" # set +e
```

Reply:
This is also in the other windows script, I think that command could be removed in both scripts.

Reply:
Yeah, I copied it around, didn't think a lot about this, I will review it, thanks!